atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaHotspottingTest.java

```diff
@@ -49,7 +49,7 @@ private static Schema getHotspottingSchema() {
     }
 
     private static Schema getIgnoredHotspottingSchema() {
-        Schema ignoredHotspottingSchema = new Schema(SCHEMA_NAME, "valid.package", Namespace.DEFAULT_NAMESPACE);
+        Schema ignoredHotspottingSchema = new Schema(SCHEMA_NAME, "valid.pkg", Namespace.DEFAULT_NAMESPACE);
```

Comment:
I got a chuckle out of this.

Reply:
Wasn't expecting this... it's fine, though is this a thing we expect from the formatter?

Reply:
Yep, it uses the jdk ast parser, which does some validation.